alerting/src/test/kotlin/org/opensearch/alerting/DocumentMonitorRunnerIT.kt

```diff
@@ -2644,6 +2645,110 @@ class DocumentMonitorRunnerIT : AlertingRestTestCase() {
         }
     }
 
+    fun `test execute monitor generates alerts and findings in a distributed way`() {
```

Comment:
plz rename this test to describe exact scenario. 

In essence, correctness should be verifed not just by using the num shards setting.

Plz add integration tests with 2 or more indices, each index containing 5 or more shards. with each shard containinig 2 or more copies(i.e. replica being present)

Reply:
renamed the test